_layouts/post.html

```diff
@@ -13,4 +13,4 @@ <h1 class="post-title" itemprop="name headline">{{ page.title }}</h1>
     {{ content }}
   </div>
 
-</article>
+</article> -->
```

Comment:
Can't this whole file be removed since it's all commented now?

Reply:
Let me sort that.. Need to get the hang of this branch business and .gitignore.